Add documentation for storage classes (#153)
purestorage/helm-charts: operator-csi-plugin/README.md

More details on using customized filesystem options can be found [here](../docs/csi-filesystem-options.md).

## PSO use of StorageClass

> Whilst there are some default storage ass definitions provided by the PSO installation, refer [here](../docs/custom-storageclasses.md) for more details on these default storage classes and how to create your own cu

Reviewer: oops
```suggestion
Whilst there are some default `StorageClass` definitions provided by the PSO installation, refer [here](../docs/custom-storageclasses.md) for more details on these default storage classes and how to create your own custom storage classes that can be used by PSO.
```

Author: Lol - fixed
